pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -636,7 +638,7 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1.PipelineRun, getPipel
 				}
 			}
 			if tasksToTimeOut.Len() > 0 {
-				logger.Infof("PipelineRun tasks timeout of %s reached, cancelling tasks", pr.Spec.Timeouts.Tasks.Duration.String())
+				logger.Infof("PipelineRun tasks timeout of %s reached, cancelling tasks", tasksTimeout)
```

Comment:
NIT: If `taskTimeout` is nil, this will not panic but the message won't be very helpful. Also, I wonder if we really need to log at `Info` level every time a timeout fires. That's also something we can fix in a separate PR though.

Reply:
Changed log level to debug! I think if `tasksTimeout` is nil, that would signify a bug in our implementation- not sure what the best way to handle that would be?